src/main/java/io/quarkus/gizmo/BytecodeCreator.java

```diff
@@ -1033,6 +1042,25 @@ default ResultHandle increment(ResultHandle toIncrement) {
         return add(toIncrement, load(1));
     }
 
+    /**
+     * Create a new switch construct.
+     * 
+     * @param value The string value to switch on
+     * @param defaultBlockConsumer
```

Comment:
Missing doc; what is this argument for?

Reply:
This argument is used to define the bytecode for the `default` label of the switch. But I'm going to change the API and add something like `StringSwitch#defaultCase()` instead.